<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>65</y>
     <width>1151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>175</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Age:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Date of Birth:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Email:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="reg_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>78</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Registration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="username_line"/>
    </item>
    <item>
     <widget class="QLineEdit" name="age_line"/>
    </item>
    <item>
     <widget class="QLineEdit" name="date_line"/>
    </item>
    <item>
     <widget class="QLineEdit" name="email_line"/>
    </item>
    <item>
     <widget class="QLineEdit" name="password_line"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>530</y>
     <width>1351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>385</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Quit From Application</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>Or Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
